src/components/ArticleController.jsx

```diff
@@ -19,17 +19,15 @@ export default class ArticleController extends FalcorController {
       let articleSlug = this.props.params.articleSlug;
       // Access data fetched via Falcor
       const articleData = this.state.data.articlesBySlug[articleSlug];
-      //console.log("Data: " + JSON.stringify(articleData));
       return (
         <div>
-          {/*
-            <div>Controller for article: {articleData.title}</div>
-            <div>Ready?: {this.state.ready ? 'true' : 'false'}</div>
-          */}
           <Article
             title={articleData.title}
+            teaser={articleData.teaser}
+            pubDate={articleData.published_at}
             html={articleData.html}
             authors={articleData.authors}
+            url={"thegazelle.org/issue/" + articleData.issueId + '/' + articleData.category + '/' + articleData.slug}
```

Comment:
It's for passing directly to `Article`'s child `SharingButtons`. It's used when dynamically constructing the sharing buttons' links. With Twitter for example, I use this prop to display the URL in the body of the tweet and make it as succinctly as possible (i.e. `thegazelle.org/issue/55/in-focus/sleep` instead of `https://www.thegazelle.org/issue/55/in-focus/sleep`). Taking it directly from the DOM, I'd get the `https://www` in the tweet text box taking up more characters and making it look less clean IMO. 

Because of this, I'd either have to construct the link like this or trim the result of the DOM's URL (which would result in more time complexity). Becuase I use this once per share button, I figured it was best to just pass it down.